README.md

```diff
@@ -344,24 +344,35 @@ Example
 
 ### 5. Localization
 
-Onfido SDKs support a number of out-of-the-box translations. However, you can add your own translations.
+The SDK supports and maintains the following 6 languages:
+
+ - English (en) 🇬🇧
+ - Spanish (es) 🇪🇸
+ - French (fr) 🇫🇷
+ - German (de) 🇩🇪
+ - Italian (it) 🇮🇹
+ - Portuguese (pt) 🇵🇹
+ 
+However, you can add your own translations.
 
 #### Android
 
 By default, custom localisation is enabled on Android. There is no configuration needed on React Native SDK to enable it.
 You could also provide custom translation for a locale that we don’t currently support, by having an additional XML strings file inside your resources folder for the desired locale. See [Localisation section of Android SDK repo](https://github.com/onfido/onfido-android-sdk#4-localisation) for the details.
 
 #### iOS
+There is simple configuration needed on React Native SDK to enable custom localisation.
+You could also provide custom translation for a locale that we don’t currently support
```

Comment:
```suggestion
You can also provide a custom translation for a locale that Onfido doesn't currently support.
```

Reply:
This should be the first sentence under #### iOS

Reply:
Also need to swap these 2 sentences around, otherwise doesn't really make sense

Reply:
fixed.